components/mpas-albany-landice/src/shared/mpas_li_time_average_coupled.F

```diff
@@ -0,0 +1,137 @@
+        end do
+
+        timeAccumulatedCoupled = 0.0_RKIND
+
+    end subroutine li_time_average_coupled_init
+
+!***********************************************************************
+!
+!  routine li_time_average_coupled_accumulate
+!
+!> \brief   Coupled time averager accumulation
+!> \author  Stephen Price
+!> \date    04 March 2025
+!> \details
+!>  This routine accumulated the coupled time averaging fields
+!
+!-----------------------------------------------------------------------
+    subroutine li_time_average_coupled_accumulate(timeAveragingPool, geometryPool, meshPool)
+            
+        use li_setup 
+
+        type (mpas_pool_type), intent(inout) :: timeAveragingPool
+        type (mpas_pool_type), intent(in) :: geometryPool
+        type (mpas_pool_type), intent(in) :: meshPool
+
+        real (kind=RKIND), dimension(:), pointer :: bareIceAblation, avgBareIceAblation, &
+                                                    calvingThickness, avgCalvingThickness, &
+                                                    faceMeltingThickness, avgFaceMeltingThickness
+        integer, pointer :: nCells
+
+        real (kind=RKIND), pointer :: timeAccumulatedCoupled, rhoi, deltat
+
+        integer :: iCell
+
+        call mpas_pool_get_dimension(meshPool, 'nCells', nCells)
+        call mpas_pool_get_config(liConfigs, 'config_ice_density', rhoi)
+        call mpas_pool_get_array(meshPool, 'deltat', deltat)
+        call mpas_pool_get_array(timeAveragingPool, 'timeAccumulatedCoupled', timeAccumulatedCoupled)
+        call mpas_pool_get_array(geometryPool, 'bareIceAblation', bareIceAblation)
+        call mpas_pool_get_array(geometryPool, 'calvingThickness', calvingThickness)
+        call mpas_pool_get_array(geometryPool, 'faceMeltingThickness', faceMeltingThickness)
+        call mpas_pool_get_array(timeAveragingPool, 'avgBareIceAblation', avgBareIceAblation)
+        call mpas_pool_get_array(timeAveragingPool, 'avgCalvingThickness', avgCalvingThickness)
+        call mpas_pool_get_array(timeAveragingPool, 'avgFaceMeltingThickness', avgFaceMeltingThickness)
+
+        do iCell = 1, nCells
+
+           avgBareIceAblation(iCell) = ( avgBareIceAblation(iCell) * timeAccumulatedCoupled &
+                                               + bareIceAblation(iCell) * deltat ) / ( timeAccumulatedCoupled + deltat )
+
+           avgCalvingThickness(iCell) = ( avgCalvingThickness(iCell) * timeAccumulatedCoupled &
+                                                + calvingThickness(iCell) * deltat ) / ( timeAccumulatedCoupled + deltat ) &
+                                                * rhoi / deltat
+
+           avgFaceMeltingThickness(iCell) = ( avgFaceMeltingThickness(iCell) * timeAccumulatedCoupled &
+                                                + faceMeltingThickness(iCell) * deltat ) / ( timeAccumulatedCoupled + deltat ) &
+                                                * rhoi / deltat
```

Comment:
same as previous

Reply:
fixed as noted above for avgCalvingThickess